mskokova/01_HTML/task_02.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>  
+<HTML> 
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>HOW-TO: Программа на Python</TITLE>
+	</HEAD> 
+	<BODY> 
+		<H1>HOW-TO: Программа на Python, Часть 3</H1>
+		<EM><UL>
```

Comment:
tag em should be in each li tag
see on slides, how to formatting inline tags

Reply:
Got it. Thanks. 
Should i correkt the mistakes and send homework again?

Reply:
> Got it. Thanks.
> Should i correkt the mistakes and send homework again?

need to do changes in same files and commit them, i will see this correction